persistence/scheduled-tasks/schedule-task-via-commandline.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: schedule task via command line
+    namespace: persistence/scheduled-tasks
+    author: 0x534a@mailbox.org
+    scope: function
+    att&ck:
+      - Persistence::Scheduled Task/Job::Scheduled Task [T1053.005]
+    examples:
+      - 79cde1aa711e321b4939805d27e160be:0x401440
+  features:
+    - and:
+        - match: create process
+        - or:
+          - and:
+            - string: /schtasks/i
+            - string: //create /i
```

Comment:
`/` should be escaped?

Reply:
Based on my understanding and on the current capa implementation this works as expected without escaping the `/` (see [capa code](https://github.com/fireeye/capa/blob/master/capa/features/__init__.py#L129)).

Reply:
right, my bad